segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/stat/AutoScaleProcessor.java

```diff
@@ -265,6 +280,7 @@ private void writeRequest(AutoScaleEvent event, Runnable successCallback) {
 
     void report(String streamSegmentName, long targetRate, long startTime, double twoMinuteRate, double fiveMinuteRate, double tenMinuteRate, double twentyMinuteRate) {
         log.info("received traffic for {} with twoMinute rate = {} and targetRate = {}", streamSegmentName, twoMinuteRate, targetRate);
+        cache.get(streamSegmentName);
```

Comment:
Why?

Reply:
The get method updates the latest access time for the contents. In case of steady traffic below target rate, neither the upscale nor downscale process occurs.  As a result, we have cache expiry and needless downscaling. This line prevents expiry when there is steady traffic.